### ISSUE TITLE
links to design page are broken

### ISSUE THREAD

**Issue by nicovince** [JuulLabs-OSS/mcuboot#803]
Since commit fe03109ab1356583f95c493259dc107ce1249d3e docs/design.md has been renamed to scripts/imgtool/design.md breaking in the process all links to docs/design.md

```
git grep design.md
docs/imgtool.md:requirements of this option, see the [design](design.md) document.
docs/index.md:- [Bootloader design](design.md)
docs/readme-zephyr.md:Please see the [design document](design.md) for documentation on the design
docs/signed_images.md:included in the bootloader, see the [design](design.md) document.
```

**Comment by utzig** [comment]
Thanks for reporting!

**Comment by utzig** [comment]
Fixed by #804
